### PR TITLE
Add Sovereign Cloud Stack as adopter of ORT

### PR DESCRIPTION
**Pull request by itrich**
We're currently in the progress of adopting ORT for our open-source cloud stack project and are happy to add us to this list.

Signed-off-by: Eduard Itrich <eduard@itrich.net>

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ This really helps the project to gain momentum and credibility. It's a small con
 
 | Organization | Contact | Description of Use |
 |--------------|---------|--------------------|
+| Sovereign Cloud Stack | https://scs.community/ | Currently in the process of evaluating how to generate a complete SBOM of our stack and secure the supply chain. We plan to both use ORT to analyze our reference implementation as well as to add ORT to our [recommended operator's toolset]([https://scs.community/about/](https://scs.community/about/#technological-vision)).  |
```
Comment (sschuberth) [inline ADOPTERS.md:8]: Something about the link syntax for the "recommended operator's toolset" seems to be wrong.

**Comment by sschuberth** [comment]
Please also rebase this onto latest `origin/main` to benefit from some test fixes / updates.

**Comment by sschuberth** [comment]
Thanks @itrich for the update! Now for the sake of maintaining a clean Git history, please squash all your commits into one (the first one).

PS: Instead of catching up with upstream changes via a merge, please *rebase* onto latest `origin/main`.

**Comment by sschuberth** [comment]
> Now for the sake of maintaining a clean Git history, please squash all your commits into one (the first one).

Thanks for the squash @itrich, however now we have two commit messages in one, and two sign-offs. Please remove this bottom part from the commit message:

```
Fix broken reference to our website

Signed-off-by: Eduard Itrich <eduard@itrich.net>
```

PS: Don't forget to click the button next to the reviewer to re-request a review after you're finished with your changes.

**Comment by sschuberth** [comment]
@itrich was the closing of this intended?

**Comment by itrich** [comment]
> @itrich was the closing of this intended?

Sorry, not at all. I cleaned up my GitHub namespace this morning and forgot about the open PR here.

**Pull request closed without merging**